rxtools/source/lib/console.c

```diff
@@ -100,7 +100,7 @@ void ConsoleShow(){
 	DrawString(tmpscreen, consoletitle, ConsoleX + BorderWidth + 2*CHAR_WIDTH, ConsoleY + (titlespace-CHAR_WIDTH)/2 + BorderWidth, TextColor, BackgroundColor);
 	
 	char tmp[256], *point;
-	if(findCursorLine < MAXLINES) point = &console[0];
+        if(findCursorLine() < MAXLINES) point = &console[0];
```

Comment:
MAXLINES as for sure smaller than findCursorLine address, so condition never met.